Tests/Fluent.Tests/ResourceManager/SubscriptionsTests.cs

```diff
@@ -65,5 +68,31 @@ public async Task CanGetByIdAsync()
                 Assert.Equal(subscription.SubscriptionId.ToLowerInvariant(), azure.SubscriptionId.ToLowerInvariant());
             }
         }
+
+        [Fact(Skip = "Util to generate missing regions")]
+        public void GenerateMissingRegion()
```

Comment:
Should we skip this test case?

Reply:
It is actually util code to faciliate generating code when azure should add new region in the future.

It is no use in replay mode, it had to run in live. Hence the SKIP.